lib/speakingurl.js

```diff
@@ -332,6 +332,43 @@
             'ޯ': 'oa',
             'ް': '',
 
+            // Georgian https://www.wikiwand.com/en/Romanization_of_Georgian
```

Comment:
Why not to link just plain raw wikipedia?

https://en.wikipedia.org/wiki/Romanization_of_Georgian